.do/values-env-temp.yaml

```diff
@@ -0,0 +1,34 @@
+# environment specific values
+global:
+  github:
+    app:
+      name: ${GITHUB_APP_NAME}
+      id: "${GITHUB_APP_ID}"
+      privateKey: "${GITHUB_PRIVATE_KEY}"
+    clientId: ${GITHUB_CLIENTID}
+    clientSecret: ${GITHUB_CLIENTSECRET}
+
+  githubEnterprise:
```

Comment:
What is enterprise in this context?

Reply:
Github self-hosted, which is mapped to the codacy-api environment in the configmap.